testsuite/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+#![cfg(test)]
+
+use std::ffi::OsStr;
+
+mod run;
+
+const BASIC_BLACKLIST: [&str; 8] = [
+	// those use unsupported i32_trunc_sat_* instructions
+	"binary-leb128.wast",
+	"conversions.wast",
+	// those use multi-value for blocks which is unsupported
+	"block.wast",
+	"func.wast",
+	"if.wast",
+	"loop.wast",
+	"fac.wast",
+	"br.wast",
```

Comment:
This is really unfortunate that we have to blacklist those. This may even render this PR as a regression in testing, because we may skip important test cases and what we get in return is potentially some new test cases in other files.

Reply:
The changes required to make almost all of them work are rather minor: Fix multi-value support. I think changing `BlockType` would suffice to get that working. 

I wanted to do that as a follow up but I can do it in this PR, too. It is a breaking change, though.